src/Illuminate/Http/RedirectResponse.php

```diff
@@ -21,7 +21,7 @@ class RedirectResponse extends BaseRedirectResponse {
 	/**
 	 * The session store implementation.
 	 *
-	 * @var \Illuminate\Session\Store
+	 * @var \Illuminate\Session\StoreInterface
```

Comment:
I revert this due the concern @taylorotwell raised regarding the dependency on Symfony/HttpFoundation.